### PR TITLE
Improvements for transaction test cases

### PR DESCRIPTION
**Pull request by anupama-pathirage**
This PR depends on https://github.com/ballerinalang/ballerina-parent/pull/94

Fixes https://github.com/ballerinalang/ballerina/issues/4294

With this PR Distributed transaction tests are added.

### CODE REVIEW

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java [hunk 1]
```diff
@@ -98,12 +121,12 @@ public static boolean deleteDirectory(File directory) {
      * Delete all the files and sub directories which matches given prefix in a given directory.
      *
      * @param directory Directory which contains files to delete.
-     * @param prefix    Prefix for matching files to delete.
+     * @param affix    Prefix for matching files to delete.
```
Comment (chamil321) [inline modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java]: Shall we change the param description according to the affix?

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java [hunk 2]
```diff
@@ -64,19 +65,41 @@ public static void initDatabase(String dbDirectory, String dbName, String sqlFil
...
+    public static void initH2Database(String dbDirectory, String dbName, String sqlFile) {
+        Connection connection = null;
+        Statement st = null;
+        try {
+            Class.forName("org.h2.Driver");
+            String jdbcURL = "jdbc:h2:file:" + dbDirectory + dbName;
+            connection = DriverManager.getConnection(jdbcURL, "sa", "");
+            String sql = readFileToString(sqlFile);
+            String[] sqlQuery = sql.trim().split("/");
+            Statement st1 = connection.createStatement();
```
Comment (chamil321) [inline modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java]: Shall we rename st1 variable name to a meaningful name?

Comment (anupama-pathirage) [reply modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java]: Fixed.

File: modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java (hunk 1 above)
Comment (anupama-pathirage) [reply modules/ballerina-test/src/test/java/org/ballerinalang/test/utils/SQLDBUtils.java]: Fixed.

**Pull request merged**